.stylelintrc.json

```diff
@@ -5,5 +5,15 @@
   "extends": [
     "stylelint-config-standard",
     "stylelint-config-styled-components"
-  ]
+  ],
+  "rules": {
+    "value-keyword-case": [
+      "lower",
+      {
+        "ignoreKeywords": [
+          "dummyValue"
```

Comment:
The latest version of stylelint is incompatible with the styled-components processor, because the processor turns mixins into `dummyValue`, but stylelint forbids using camelCase in CSS. Here we explicitly ignore only this one value, the rule is still enabled for all other values.